lib/cisco_node_utils/vxlan_global.rb

```diff
@@ -29,17 +30,19 @@ def initialize(instantiate=true)
     end
 
     def enable
-      config_set('vxlan_global', 'feature', state: '')
+      Feature.fabric_enable unless Feature.fabric_enabled?
+      config_set('feature', 'fabric_forwarding', state: '')
     end
 
     def disable
-      config_set('vxlan_global', 'feature', state: 'no')
+      config_set('feature', 'fabric_forwarding', state: 'no') if
+        Feature.fabric_enabled?
       dup_host_mac_detection_default
     end
 
     # Check current state of the configuration
     def self.enabled
```

Comment:
This should be removed and the callers should be updated to use `Feature.fabric_forwarding` instead.